sound/soc/sof/ipc4-priv.h

```diff
@@ -13,18 +13,33 @@
 #include <sound/sof/ext_manifest4.h>
 #include "sof-priv.h"
 
+/* The DSP window indices are fixed */
+#define SOF_IPC4_OUTBOX_WINDOW_IDX	1
```

Comment:
We always get the inbox offset with the call `snd_sof_dsp_get_mailbox_offset()`, so I thought it would be just awkward to have a new define, which is unused?